pkgs/applications/networking/instant-messengers/matrix-commander/default.nix

```
@@ -0,0 +1,36 @@
+{ pkgs, fetchFromGitHub, lib }:
+
+pkgs.stdenv.mkDerivation {
+  name = "matrix-commander";
```

```suggestion
  pname = "matrix-commander";
  version = "unstable-XXXX-XX-XX"
```

replace XXXX-XX-XX with the date of the commit.